xml/art_suseconnect_tracking.xml

```diff
@@ -0,0 +1,405 @@
+]>
+<article xml:id="article-suseconnect-tracking"
+         xmlns="http://docbook.org/ns/docbook"
+         version="5.0"
+         xml:lang="en"
+         xmlns:xi="http://www.w3.org/2001/XInclude"
+         xmlns:xlink="http://www.w3.org/1999/xlink">
+ <title>Tracking active systems with &suseconnect;</title>
+ <titleabbrev>Tracking active systems with &suseconnect;</titleabbrev>
+ <info><productname>&productnamex;</productname>
+  <productnumber>&productnumberx;</productnumber><date>
+<?dbtimestamp format="B d, Y" ?></date>
+  <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+   <dm:bugtracker>
+    <dm:url>https://bugzilla.suse.com/enter_bug.cgi</dm:url>
+    <dm:component>Documentation</dm:component>
+    <dm:assignee>tbazant@suse.com</dm:assignee>
+   </dm:bugtracker>
+   <dm:editurl>https://github.com/SUSE/doc-unversioned/blob/main/xml/</dm:editurl>
+   <dm:translation>yes</dm:translation>
+  </dm:docmanager>
+  <abstract>
+   <para>
+    This document document describes what &suseconnect; is and how the &scc; (SCC) and Repository
+    Mirroring Tool (RMT) use it to gather information about active systems and their hardware
+    environment.
+   </para>
+  </abstract>
+ </info>
+ <sect1 xml:id="suseconnect-whatis">
+  <title>What is &suseconnect;?</title>
+
+  <para>
+   &suseconnect; is a tool to register SLE-based operating systems with SCC or RMT. Registered
+   systems receive updates to installed packages, for example, to increase security of the operating
+   system. With &suseconnect;, you can additionally register extensions and modules that extend the
+   base system functionality.
+  </para>
+
+  <tip>
+   <para>
+    This document does not cover general usage of &suseconnect;. To find more basic information
+    about how to register your system, modules, and extensions with &suseconnect;, refer to
+    <link xlink:href="https://documentation.suse.com/sles/html/SLES-all/cha-register-sle.html"/>.
+   </para>
+  </tip>
+
+  <para>
+   With newer versions of &suseconnect;, each registered host contacts SCC or RMT automatically on a
+   daily basis. It sends unique identification of the host and description of its hardware
+   environment.
+  </para>
+ </sect1>
+ <sect1 xml:id="suseconnect-tracking-requirements">
+  <title>Requirements</title>
+
+  <para>
+   In order to use the extended &suseconnect; functionality, you need to:
+  </para>
+
+  <itemizedlist>
+   <listitem>
+    <para>
+     Run a supported and registered SLE-based host with all update patches applied.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Verify that the host system includes the <command>&suseconnect;</command> command version
+     0.3.33 or higher.
+    </para>
+   </listitem>
+  </itemizedlist>
+ </sect1>
+ <sect1 xml:id="suseconnect-tracking-how-it-works">
+  <title>How tracking inactive systems works</title>
+
+  <para>
+   We extended &suseconnect; with the new option <option>--keep-alive</option>. The command
```

Comment:
Here and in the rest of the document, notice that the option is `--keepalive`, not `--keep-alive`.